privacy_policy/en/privacy-notice-product-browser-extensions.md

```diff
@@ -0,0 +1,58 @@
+# Adguard Browser Extensions Privacy Notice
+
+> **Keynote:** Adguard does not share/sell any of your personal data. We are proud to say that we are one of the fighters for users privacy and we are strongly committed to this principle and to being as transparent as possible.
+
+Our privacy policy lists the information that is collected and the processing applied to it. All the info collected includes no more than is crucial to provide full functionality of Adguard products, and is never shared with any third parties. We do not collect anything for tracking purposes and take all necessary technical, administrative and physical measures to protect the information we get.
+
+Please read our privacy policy to learn what kind of information we collect via Adguard extensions and how we store and use it afterwards.
+
+## What we collect when you use Adguard browser extensions
+### Adguard extension installation
+Adguard extension connects to its server just after installation or update. When it happens, the following information is sent:
+* Unique extension ID.
+* Extension version.
+* Browser language.
+* «Show acceptable ads» setting value.
+
+We do this with a single purpose: to know the number of unique installations of our extension. This information is stored for 24 hours and then it is deleted.
+
+Also Adguard makes a request to our server in order to get user's country code. The only purpose of this is to enable proper language-specific filter.
+
+### Ad Blocker filters updates check
+Adguard extension connects to its servers to check ad blocker filters updates. When it happens, the following information is sent:
+* Ad blocker filters identifiers.
+* Current ad blocker filters versions.
+* Browser extension type.
+* Extension version.
+
+### Browsing Security site check
+If you enable «Phishing and malware protection» option, Adguard extension checks every web site before you visit it. We use Lookup API technology for this purpose, and the information about visited website is sent in form of hash prefixes. This doesn’t allow us to determine which website is being visited.
+ 
+### Statistics for ad filters usage
+If you enable the [«Send statistics for ad filters usage»](https://kb.adguard.com/en/general/filter-rules-statistics) option, the following information is sent periodically:
```

Comment:
Let's emphasize, that this option is **disabled by default**. We get this information from volunteers only